test-cases/longevity/longevity-large-partition-2days.yaml

```diff
@@ -0,0 +1,41 @@
+test_duration: 3000
+
+bench_run: true
+
+prepare_write_cmd:  ["scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=8 -clustering-row-count=50000 -clustering-row-size=uniform:100..5120 -concurrency=250 -rows-per-request=2000 -timeout=180s -connection-count 150",
+                     "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=8 -clustering-row-count=50000 -partition-offset=8 -clustering-row-size=uniform:100..5120 -concurrency=250 -rows-per-request=2000 -timeout=180s -connection-count 150",
+                     "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=8 -clustering-row-count=50000 -partition-offset=16 -clustering-row-size=uniform:100..5120 -concurrency=250 -rows-per-request=2000 -timeout=180s -connection-count 150",
+                     "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=8 -clustering-row-count=50000 -partition-offset=24 -clustering-row-size=uniform:100..5120 -concurrency=250 -rows-per-request=2000 -timeout=180s -connection-count 150",
+                     "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=7 -clustering-row-count=50000 -partition-offset=32 -clustering-row-size=uniform:100..5120 -concurrency=250 -rows-per-request=2000 -timeout=180s -connection-count 150",
+                     "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=7 -clustering-row-count=50000 -partition-offset=39 -clustering-row-size=uniform:100..5120 -concurrency=250 -rows-per-request=2000 -timeout=180s -connection-count 150",
+                     "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=7 -clustering-row-count=50000 -partition-offset=46 -clustering-row-size=uniform:100..5120 -concurrency=250 -rows-per-request=2000 -timeout=180s -connection-count 150",
+                     "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=7 -clustering-row-count=50000 -partition-offset=53 -clustering-row-size=uniform:100..5120 -concurrency=250 -rows-per-request=2000 -timeout=180s -connection-count 150"
```

Comment:
Add much more partitions.
instead of 7 or 8 each, make it 100 each.
If the prepare finish less than 2 hours, increase it to even more.

Reply:
fixed